conf/azurebatch.config

```
@@ -8,6 +8,7 @@ params {
     storage_name      = null
     storage_key       = null
     storage_sas       = null
+    storage_duration  = null
```

```suggestion
    storage_duration  = '48h'
```
Based on the default value given below, but no azure user myself.